docs/flow/modin/data_management/factories.rst

```diff
@@ -0,0 +1,19 @@
+Factories Module Description
+""""""""""""""""""""""""""""
+
+Brief description
+'''''''''''''''''
+Modin has many execution backends. Calling any DataFrame API function will end up in some backend-specific method. Factories module role is to route IO function calls from the API level to its actual backend-specific implementations.
+
+Backend representation via Factories
```

Comment:
Both here and in `Engine dispatcher` section you refer to IO operations only. That is not fully true. Factories are responsible for routing of all function calls from the API level to a specific backend. I think we should also note about a frame and a query compiler classes that are held by conforming IO class. These frame and query compiler classes are responsible for routing of function calls different from IO.

Reply:
`EngineDispatcher` is responsible only for IO calls, you can walk through [its methods](https://github.com/modin-project/modin/blob/a613f2d10a59c7a23fbae7b899c3a6c21d2c0c35/modin/data_management/factories/dispatcher.py#L50) to be sure. 

As you correctly mentioned, the rest of the calls are handled by the query compiler, of the selected backend (not by `EngineDispatcher` or factory). Going to add a note in docs about it.

Reply:
Strictly saying, yes, the rest of the calls are handled by the query compiler that is selected through the following flow: EngineDispatcher -> Factory -> IO -> QC. That should be highlighted.

Reply:
> `EngineDispatcher` is responsible only for IO calls, you can walk through [its methods](https://github.com/modin-project/modin/blob/a613f2d10a59c7a23fbae7b899c3a6c21d2c0c35/modin/data_management/factories/dispatcher.py#L50) to be sure.

`from_pandas`, `from_arrow`, `from_non_pandas` :)

Reply:
clarified qc role in dispatching function calls at the last commit

Reply:
I would prefer to highlight the Factories role in initialization of the query compiler a bit more. Something like this:
`Modin has many execution backends. Calling any DataFrame API function will end up in some backend-specific method. This responsibility lays on QueryCompiler that is initialized when creating a DataFrame through dispatching of required factory.`

Reply:
rephrased your suggestion a bit and added it to the docs

Reply:
Rephrased a bit as well :)